Forwarded: https://github.com/luceneplusplus/LucenePlusPlus/pull/64
Author: Gianfranco Costamagna <costamagnagianfranco@yahoo.it>
Description: Disabling tests until we find a proper way for building and including gtest
 In the next release this might become possible, after commit 83356d9681fb52
 https://github.com/luceneplusplus/LucenePlusPlus/commit/83356d9681fb52a4c190d2cf033279006c1bf57d
Last-Update: 2014-08-25
--- a/CMakeExternal.txt
+++ b/CMakeExternal.txt
@@ -1,5 +1,6 @@
 # Enable ExternalProject CMake module
 include(ExternalProject)
+find_package(Subversion REQUIRED)
 
 # main directory for external projects
 set_directory_properties(PROPERTIES EP_PREFIX ${CMAKE_CURRENT_BINARY_DIR}/ThirdParty)
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -49,7 +49,6 @@
 # bootstrap
 ####################################
 
-find_package(Subversion REQUIRED)
 find_package(Threads REQUIRED)
 find_package(Boost COMPONENTS
   date_time
@@ -114,13 +113,13 @@
   ${CMAKE_CURRENT_BINARY_DIR}/include
 )
 
-include(CMakeExternal.txt)
-enable_testing()
+#include(CMakeExternal.txt)
+#enable_testing()
 
 add_subdirectory(src/core)
 add_subdirectory(src/contrib)
 #add_subdirectory(src/demo)
-add_subdirectory(src/test)
+#add_subdirectory(src/test)
 
 #################################
 # install pkg-config file
